apps/web/content/docs/en/core/fees.mdx

```diff
@@ -5,117 +5,107 @@ description:
+- [How to Request Optimal Compute](/developers/guides/advanced/how-to-request-optimal-compute)
 
-See the
-[How to Request Optimal Compute](/developers/guides/advanced/how-to-request-optimal-compute)
-guide for more details on compute unit usage.
-
-### Compute Unit Price
+### Compute unit price
 
 The compute unit price is an optional amount, specified in micro-lamports, that
-you pay for each compute unit requested. This price is used to calculate the
-prioritization fee for your transaction.
+you pay for each compute unit requested. You can think of the compute unit price as a "tip" for the validator to encourage them to prioritize your transaction. 
 
 <Callout>1,000,000 micro-lamports = 1 lamport</Callout>
 
-Use these resources to get real-time recommendations on the current compute unit
-price:
-
-| Provider                                | Priority Fee API                                                                  |
-| --------------------------------------- | --------------------------------------------------------------------------------- |
-| [Helius](https://www.helius.dev/)       | [Documentation](https://docs.helius.dev/solana-apis/priority-fee-api)             |
-| [QuickNode](https://www.quicknode.com/) | [Documentation](https://marketplace.quicknode.com/add-on/solana-priority-fee)     |
-| [Triton](https://triton.one/)           | [Documentation](https://docs.triton.one/chains/solana/improved-priority-fees-api) |
-
 See the
 [How to Use Priority Fees](/developers/guides/advanced/how-to-use-priority-fees)
 guide for more details on priority fees.
 
-### Calculate Prioritization Fee
+### Calculate prioritization fee
 
-The
-[prioritization fee](https://github.com/anza-xyz/agave/blob/v2.2.14/compute-budget/src/compute_budget_limits.rs#L47-L48)
-is calculated as:
+Use the following formula to calculate your prioritization fee:
 
 **Prioritization Fee = Compute Unit Limit × Compute Unit Price**
 
-The recommended approach for setting priority fees is to first
+See relevant code regarding calculating [prioritization fees](https://github.com/anza-xyz/agave/blob/v2.2.14/compute-budget/src/compute_budget_limits.rs#L47-L48).
+
+#### Calculate compute unit limit
+The recommended approach for calculating the `Compute Unit Limit` is to first
 [simulate](/developers/guides/advanced/how-to-request-optimal-compute) the
 transaction to estimate the required compute units. Then, add a 10% safety
-margin to this estimate and use the resulting value as the `Compute Unit Limit`.
+margin to this estimate.
 
-The
-[transaction priority](https://github.com/anza-xyz/agave/blob/v2.2.14/core/src/banking_stage/transaction_scheduler/receive_and_buffer.rs#L646),
-which determines how it's prioritized relative to other transactions, is
-calculated using the following formula:
+{/* Compute unit limit is in backticks here, but nowhere else. Not sure whether it's appropriate here 
+or if it should be included everywhere*/}
```

Comment:
Yeah, let's remove the backticks for consistency and we can think about the general rule for these things